### DISCUSSION TITLE
roll dice in Macro

### DISCUSSION THREAD

**Discussion by LeJoe424** [Mezryss/FVTT-Genesys#93] [Q&A] [answered]
Hello !

I'm GM a game with your system, and I would like to create some macros to roll specific rolls.
Problem is, I don't know how to call for Genesys dice, like I would for d6s or d8s.

Thank you in advance !
And thank you for the system.

**Comment by Assembling-Kings** [comment, marked as answer]
The dice types are as follow:

| Dice | Denomination |
| --- | --- |
| Ability (Green) | dA |
| Proficiency (Yellow) | dP |
| Boost (Blue) | dB |
| Difficulty (Purple) | dI |
| Challenge (Red) | dC |
| Setback (Black) | dS |

However, if you roll them directly (through the chat or by using the Roll function) and send them to chat the result won't use the custom chat message. The work to implement that is being tracked on https://github.com/Mezryss/FVTT-Genesys/issues/10
